core/src/services/oss/core.rs

```diff
@@ -186,6 +187,53 @@ impl OssCore {
         Ok(req)
     }
 
+    /// Oss append object request
+    ///
+    /// # Note
+    ///
+    /// This request is used to append data to an existing object or create an appendable object.
+    /// So you must set the `append` and `position` header.
```

Comment:
Who is `you`? Please make it clear.

Reply:
🤣 Should be `we`, the service developer.